pulsar-package-management/bookkeeper-storage/src/main/java/org/apache/pulsar/packages/management/storage/bookkeeper/BookKeeperPackagesStorage.java

```diff
@@ -92,12 +93,14 @@ private URI initializeDlogNamespace() throws IOException {
             ledgersRootPath = metadataServiceUri.getPath();
         } else {
             ledgersRootPath = configuration.getPackagesManagementLedgerRootPath();
-            ledgersStoreServers = configuration.getZookeeperServers();
+            ledgersStoreServers = configuration.getMetadataStoreUrl();
+            if (ledgersStoreServers.startsWith(ZK_SCHEME_IDENTIFIER)) {
```

Comment:
can we parse it as URI and remove the authority ? I think it's not a good idea to add this exact replacement since there's no support for other protocols